linera-core/src/client/mod.rs

```diff
@@ -610,6 +601,16 @@ impl<P: 'static, S: Storage> ChainClient<P, S> {
         Unsend::new(self.state().inner.map(|state| state.pending_blobs()))
     }
 }
+#[cfg(with_testing)]
+impl<P: 'static, S: Storage> ChainClient<P, S> {
+    pub fn block_hash(&self) -> Option<CryptoHash> {
+        self.state().tip().1
```

Comment:
This `.1` is a little bit of a regression. Can we keep the individual getters and can we define a `struct Tip` with proper field names?